dags/openshift_nightlies/config/install/hypershift/none-type.json

```diff
@@ -0,0 +1,17 @@
+{
+    "cluster_install_method": "osd",
+    "ocm_cli_version": "master",
+    "ocm_environment": "stage",
+    "rosa_environment": "staging",
+    "rosa_expiration_time": "2880 minutes",
+    "openshift_worker_count": 9,
+    "openshift_network_type": "OpenShiftSDN",
+    "openshift_worker_instance_type": "m5.2xlarge",
+    "machineset_metadata_label_prefix": "machine.openshift.io",
+    "openshift_workload_node_instance_type": "m5.2xlarge",
```

Comment:
are we actually using the workload node for the management cluster?

Reply:
I will remove workload spec.